test/test_docker_helpers.py

```diff
@@ -36,15 +37,15 @@ def test_get_params_no_cache(self):
         self._client.inspect_container.return_value = self._params
         params = self._helper.get_params(self._cid)
         self._client.inspect_container.assert_called_once_with(self._cid)
-        self.assertDictEqual(params, self._params)
+        self.assertDictEqual(params, self._helper.rename_keys_to_lower(copy(self._params)))
```

Comment:
this `rename_keys_to_lower` should not be used in tests at all, IMO. For input params, changing to lower is part of the test. For expected values, they should be just already lower case, instead of calling some conversion code and misleading about what is the actual expected result